server/accounts.go

```diff
@@ -4281,25 +4372,33 @@ func newTransform(src, dest string) (*transform, error) {
 
 		nphs := 0
 		for _, token := range dtokens {
-			tp, nb, err := placeHolderIndex(token)
+			tranformType, transformArgWildcardIndexes, transfomArgInt, transformArgString, err := indexPlaceHolders(token)
 			if err != nil {
 				return nil, err
 			}
-			if tp[0] >= 0 {
+
+			if tranformType == NoTransform {
+				{
```

Comment:
I think Matthias' comment was that this "{" inside the if statement is not needed. Please remove.

Reply:
ooh ok I did misunderstood yeah cleaned it up - thanks